packages/conf-unwind/conf-unwind.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+maintainer: "David Allsopp <david@tarides.com>"
+homepage: "https://www.nongnu.org/libunwind/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+authors: ["David Mosberger, Dave Watson, et al"]
+license: "MIT"
+build: [["pkg-config" "libunwind"]]
+depends: ["conf-pkg-config" {build}]
+depexts: [
+  ["libunwind-dev"] {os-family = "debian"}
+  ["libunwind-devel"] {os-distribution = "centos"}
+  ["libunwind-devel"] {os-distribution = "rhel"}
+  ["libunwind-devel"] {os-distribution = "fedora"}
+  ["libunwind-dev"] {os-distribution = "alpine"}
+  ["libunwind"] {os-distribution = "homebrew" & os = "macos"}
```

Comment:
In my experience, building on macOS only require to `brew install libunwind-headers`.

Reply:
```suggestion
  ["libunwind-headers"] {os-distribution = "homebrew" & os = "macos"}
```